i18n-helpers/src/lib.rs

```diff
@@ -172,14 +185,14 @@ pub enum Group<'a> {
 ///         Group::Skip(&[
 ///             (1, Event::Start(Tag::List(None))),
 ///             (1, Event::Start(Tag::Item)),
-///         ]),
+///         ], Default::default()),
```

Comment:
I much prefer `GroupExtra::default()` over `Default::default()` — using the final type instead of the trait shows me what I'm getting.

Reply:
Agreed!  Okay, switched.